mix.exs

```diff
@@ -5,11 +5,11 @@ defmodule HashRing.Mixfile do
     [
       app: :hash_ring_ex,
       version: "1.1.2",
-      elixir: "~> 1.0.0 or ~> 0.15.1",
+      elixir: "~> 1.1",
```

Comment:
@jordan0day If you change it from "~> 1.0.0 or ~> 0.15.1" to "~> 1.0 or ~> 0.15.1", then you maintain backwards compatibility and it doesn't throw a warning at you.